pkg/agent/nodeportlocal/portcache/port_table_linux.go

```diff
@@ -32,9 +32,9 @@ const (
 	// protocol (as a means to reserve the port for this protocol), but no
 	// NPL rule has been installed for it.
 	stateOpen protocolSocketState = iota
-	// stateInUse means that a listening socket has been opened AND a NPL
-	// rule has been installed.
-	stateInUse
+	// stateInUse is a shared constant defined inside port_table.go, here we use a
+	// blank identifier to skip its value.
+	_
```

Comment:
It's a little overkill to distribute the constants in 3 files. I think even if you don't make this change, the issue has been resolved now? Even if we want to move these constants, I would just move them to port_table.go, not using `stateOpen` and `stateClosed` in port_table_windows.go is not a problem at all. But leaving them unmodified is perhaps better as it's no longer a problem

Reply:
okay sure, actually unused error was displayed by github check when all the constants were in port_table.go so I distributed these constants in two files, but if is acceptable then I will move them back to port_table.go.

Reply:
OK, I didn't know golangci-lint considered this as deadcode when only one platform uses it. Guess it was put in the way in the beginning was because of this. I feel better to leave them unmodified, which looks neater than defining 3 files and introducing a placeholder.